docs/resources/aws_vpc_subnets.md

```diff
@@ -0,0 +1,98 @@
+---
+title: About the aws_vpc_subnets Resource
+---
+
+# aws_vpc_subnets
+
+Use the `aws_vpc_subnets` InSpec audit resource to test properties of some or all subnets.
+
+Subnets are networks within a VPC that can have their own block of IP address's and ACL's.
+VPCs span across all availability zones in AWS, while a subnet in a VPC can only span a single availability zone.
+Separating IP addresses allows for protection if there is a failure in one availability zone.
+
+<br>
+
+## Syntax
+
+An `aws_vpc_subnets` resource block uses an optional filter to select a group of subnets and then tests that group.
+
+    # Test all subnets within a single vpc
+    describe aws_vpc_subnets.where(vpc_id: 'vpc-12345678') do
+      its('subnet_id') { should include ['subnet-12345678', 'subnet-98765432'] }
```

Comment:
Below, the property is plural (which I think reads better) - `subnet_ids`, not `subnet_id`.